core-bundle/src/Resources/views/Error/invalid_request_token.html.twig

```diff
@@ -11,5 +11,5 @@
 {% endblock %}
 {% block explain %}
     <p>{{ 'XPT.tokenExplainOne'|trans|raw }}</p>
-    <p>{{ 'XPT.tokenExplainTwo'|trans|raw }}</p>
+    <p>{{ 'XPT.tokenExplainTwo'|trans|format('https://to.contao.org/support')|raw }}</p>
```

Comment:
Shouldn't we able to pass those as arguments to `trans()`?

Reply:
Doesn‘t work for `sprintf()` placeholders. See e.g. https://github.com/contao/contao/blob/4.13/installation-bundle/src/Resources/views/configuration_error.html.twig.

Reply:
Yes, but these are Twig templates and not Contao templates. ;)

Reply:
And? The `trans` filter uses the Symfony translator which we decorated. I've just tested it and it works perfectly fine:

`{{ 'ERR.maxval'|trans(['field', 2], 'contao_default') }}` outputs ` Field "field" must not be greater than 2! ` 🤷‍♂️

Reply:
Ah, I didn‘t try an array argument. Hang on…

Reply:
BTW, why did we not do this in the other templates (see above) as well?

Reply:
I don't know, this is the first time I see the `|format` filter :)

Reply:
Changed in cfe309d0993e5da4635c2c26ee7544545a88cb58.